### PR TITLE
fix/update to chron 030

### PR DESCRIPTION
**Pull request by scealiontach**
- fix(make): update makefile to support arm64/amd64 images
- docs: updates for 0.3.0 conventions

### CODE REVIEW

File: domains/manufacturing/guide.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Output should look something like this -
 
 ```graphql
 mutation {
-  itemManufactured(name:"rotorbladefab-20221014-001",attributes:{batchIdAttribute:"20221014-001"}) {
+  itemManufactured(externalId"rotorbladefab-20221014-001",attributes:{batchIdAttribute:"20221014-001"}) {
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:174]: foo now defineFoo
Missing colon after externalId

File: domains/manufacturing/guide.md [hunk 2]
```diff
@@ -256,7 +256,7 @@ the identity and partID were incremented.
 
 ```graphql
 mutation {
-  item(name:"rotorblade-20221014-0001",attributes:{partIdAttribute:"20221014-0001"}) {
+  item(externalId"rotorblade-20221014-0001",attributes:{partIdAttribute:"20221014-0001"}) {
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:259]: foo now defineFoo
Missing colon after externalId

File: domains/manufacturing/guide.md [hunk 3]
```diff
@@ -366,7 +366,7 @@ Here the identity of the activity can incorporate (if we chose) the partID.
 
 ```graphql
 mutation {
-  itemCertified(name:"rotorbladecert-20221014-0001") {
+  itemCertified(externalId"rotorbladecert-20221014-0001") {
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:369]: foo now defineFoo
Missing colon after externalId

File: domains/manufacturing/guide.md [hunk 4]
```diff
@@ -449,7 +449,7 @@ example the certID is the same as the partID of the rotor blade.
 
 ```graphql
 mutation {
-  certificate(name:"rotorbladecert-20221014-0001",attributes:{certIdAttribute:"20221014-0001"}) {
+  certificate(externalId"rotorbladecert-20221014-0001",attributes:{certIdAttribute:"20221014-0001"}) {
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:452]: foo now defineFoo
Missing colon after externalId

File: domains/manufacturing/guide.md [hunk 5]
```diff
@@ -171,7 +171,7 @@ Output should look something like this -
 
 ```graphql
 mutation {
-  itemManufactured(name:"rotorbladefab-20221014-001",attributes:{batchIdAttribute:"20221014-001"}) {
+  itemManufactured(externalId"rotorbladefab-20221014-001",attributes:{batchIdAttribute:"20221014-001"}) {
     correlationId,
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:175]: As above
correlationId now txId
also the format of context returned is different

File: domains/manufacturing/guide.md [hunk 6]
```diff
@@ -256,7 +256,7 @@ the identity and partID were incremented.
 
 ```graphql
 mutation {
-  item(name:"rotorblade-20221014-0001",attributes:{partIdAttribute:"20221014-0001"}) {
+  item(externalId"rotorblade-20221014-0001",attributes:{partIdAttribute:"20221014-0001"}) {
     context,
     correlationId
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:261]: As above
correlationId now txId
also the format of context returned is different

File: domains/manufacturing/guide.md [hunk 7]
```diff
@@ -366,7 +366,7 @@ Here the identity of the activity can incorporate (if we chose) the partID.
 
 ```graphql
 mutation {
-  itemCertified(name:"rotorbladecert-20221014-0001") {
+  itemCertified(externalId"rotorbladecert-20221014-0001") {
     correlationId,
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:370]: As above
correlationId now txId
also the format of context returned is different

File: domains/manufacturing/guide.md [hunk 8]
```diff
@@ -449,7 +449,7 @@ example the certID is the same as the partID of the rotor blade.
 
 ```graphql
 mutation {
-  certificate(name:"rotorbladecert-20221014-0001",attributes:{certIdAttribute:"20221014-0001"}) {
+  certificate(externalId"rotorbladecert-20221014-0001",attributes:{certIdAttribute:"20221014-0001"}) {
     context,
     correlationId
```
Comment (duncanjw) [inline domains/manufacturing/guide.md:454]: As above
correlationId now txId
also the format of context returned is different

**Review by duncanjw — CHANGES_REQUESTED** [review]
I will have a go at fixing these issues

File: domains/manufacturing/guide.md [hunk 9]
```diff
@@ -145,7 +145,7 @@ First record that there was a contractor Helicopters PLC based in Bristol.
 
 ```graphql
 mutation {
-  contractor(name:"helicoptersplc",attributes:{locationAttribute:"Bristol"}) {
+  contractor(externalId"helicoptersplc",attributes:{locationAttribute:"Bristol"}) {
     context
     correlationId
```
[the remark this answers is not in the document]
Comment (duncanjw) [reply domains/manufacturing/guide.md:150]: This also means updating the subscription in README.md

File: domains/manufacturing/guide.md [hunk 10]
```diff
@@ -15,16 +16,19 @@ high specification rotor blades, which it does so in batches.
 
 #### Modelling the Contractor Agent
 
-The `Contractor` agent has one attribute, its `Location`. In our Chronicle
-domain specification this is captured as follows -
+The `Contractor` agent has two attributes, its `CompanyName` and its `Location`.
+In our Chronicl domain specification this is captured as follows -
```
Comment (mtbc) [inline domains/manufacturing/guide.md:20]:
```suggestion
In our Chronicle domain specification this is captured as follows -
```

File: domains/manufacturing/guide.md [hunk 11]
```diff
@@ -226,191 +280,191 @@ Output should look something like this -
...
+#### Record the Manufacturing of Batch of Rotor Blades
 
-For each rotor blade produced by this activity we first record its creation.
+For each rotor blade produced by this activity, we first record its creation.
 
 **NOTE** that here we are focusing on a single rotor blade. However, in a real
-world example this process would be repeated and we would need to make sure that
-the identity and partID were incremented.
+world example, this process would be repeated and we would need to make sure that
+the `name` (in future release `external_id`) and `partId` were incremented.
```
Comment (mtbc) [inline domains/manufacturing/guide.md:309]: Is it now `externalId` in even the present release?

File: domains/manufacturing/guide.md [hunk 12]
```diff
@@ -226,191 +280,191 @@ Output should look something like this -
...
+    "defineItem": {
+      "context": "chronicle:entity:rotorblade%2Drun%2D001%2D001",
+      "txId": "630356af-01e1-42cd-9513-f52b59c70d53"
     }
   }
 }
 ```
 
-Then we assert that it `wasGeneratedBy` the relevant manufacturing activity.
+Then we assert that it `wasGeneratedBy` by the relevant manufacturing activity.
```
Comment (mtbc) [inline domains/manufacturing/guide.md:333]: Not convinced that the second "by" helps, but I don't strongly object.

File: domains/manufacturing/guide.md (hunk 11 above)
Comment (duncanjw) [reply domains/manufacturing/guide.md:309]: Correct. Will fix

File: domains/manufacturing/guide.md (hunk 12 above)
Comment (duncanjw) [reply domains/manufacturing/guide.md:333]: Reads as was generated by without it :)

**Comment by mtbc** [comment]
Also fixes needed to the other example domains but those could be in other PRs.

File: domains/manufacturing/guide.md (hunk 10 above)
Comment (duncanjw) [reply domains/manufacturing/guide.md]: Fixed

File: README.md [hunk 13]
```diff
@@ -124,7 +122,7 @@ in one of the tabs.
 ```graphql
 subscription {
   commitNotifications {
-    correlationId
+    txId
```
Comment (mtbc) [inline README.md:125]: I'll leave it to @ryanbtp to think if we want to add `stage`.

**Review by mtbc — APPROVED** [review]
Looks good to me at least.

**Review by duncanjw — APPROVED** [review]

File: README.md (hunk 13 above)
Comment (scealiontach) [reply README.md:125]: deal with it at a later pull I think.  I don't see the semantics of those documented anywhere yet (maybe I missed it).  Best not to have "what is that?" items in the examples, until we can point to the answer.

**Pull request merged**